packages/cli-platform-ios/src/tools/getBuildConfigurationFromXcScheme.ts

```diff
@@ -13,29 +13,45 @@ export function getBuildConfigurationFromXcScheme(
   sourceDir: string,
   projectInfo: IosInfo | undefined,
 ): string {
+  let xcProjects: string[] = [];
   try {
-    const xcProject = fs
+    xcProjects = fs
       .readdirSync(sourceDir)
-      .find((dir) => dir.includes('.xcodeproj'));
-
-    if (xcProject) {
-      const xmlScheme = fs.readFileSync(
-        path.join(
-          sourceDir,
-          xcProject,
-          'xcshareddata',
-          'xcschemes',
-          `${scheme}.xcscheme`,
-        ),
-        {
-          encoding: 'utf-8',
-        },
-      );
-
-      const {Scheme} = xmlParser.parse(xmlScheme);
-
-      return Scheme.LaunchAction['@_buildConfiguration'];
-    }
+      .filter((x) => x.endsWith('.xcodeproj'));
+  } catch (e: any) {
+    throw new CLIError(`Unknown error: ${e.toString()}`);
+  }
+
+  if (xcProjects.length === 0) {
+    throw new CLIError(
+      'Xcode project not found. please make sure it is a iOS project',
+    );
+  }
+
+  if (xcProjects.length > 1) {
+    throw new CLIError(
+      'Multiple Xcode projects found. we are only support one. please remove another one',
+    );
+  }
```

Comment:
I believe in that case, instead of forcing to remove the project, we should prompt user which project to use -> maybe store it in cache?

Reply:
I’m not sure whether we should prompt the user to choose one option for processing. Additionally, it seems that the current process does not support different projects. the previous code used a simple `[].find` to select the first available option, but I guess this is not the only occurrence.

this PR just corrects the error message here. If the CLI is going to support multiple projects in future, we can easily identify where we need to make changes.

additionally, I’m not familiar with this project, so I don’t know how to show a prompt to the user in this project 😂

If necessary, maybe we can raise another pull request after this one is merged? This one is for correcting error messages, and the next one will be for enhancing the DX.